spec/models/user_spec.rb

```diff
@@ -1,3 +1,4 @@
+include Rails.application.routes.url_helpers
```

Comment:
I am not sure whether you need this line or not.

Reply:
I added this line in order to use url_for method in user_rspec.rb, i tried to remove this line and Rails.application.routes.default_url_options[:host]='localhost:3000', but I get a NoMethodError.

Reply:
hmm, I think you need this line. Very good point!
Later I would move this line to RSpec config file.